.travis.yml

```diff
@@ -16,6 +16,7 @@ branches:
   only:
     - csi-v0.3
     - master
+    - release-v1.1.0
```

Comment:
@Madhu-1 do we need to remove csi-v0.3 ?

Reply:
removed